tests/CMakeLists.txt

```diff
@@ -257,7 +257,7 @@ FOREACH(_test ${_tests})
 	  | egrep -v '^--'
 	  | sed 's\#${ASPECT_BASE_DIR}\#ASPECT_DIR\#g'
 	  > ${CMAKE_CURRENT_BINARY_DIR}/output-${_test}/${_output}.notime
-	DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/output-${_test}/screen-output
+	DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/output-${_test}/${_output}
 		${CMAKE_CURRENT_BINARY_DIR}/output-${_test}/${_output}.cmp.notime
```

Comment:
Good catch -- your change looks right. I don't understand, though, why we have the second dependency. Any idea? It probably doesn't hurt much, but it looks like we're not using it here. It should be a dependency for the next target only (and it is), right?